### PR TITLE
Zenodo update

### LINKED ISSUE #159: Add authorship section to CONTRIBUTOR.md
Relates to #152, #158 

Suggestion:

>## Authorship
>
>Thank you for contributing to this repo! If you would like to be cited as an author/contributor of this project please open a PR adding yourself to the [.zenodo.json file](.zenodo.json) along with your [ORCID ID](https://orcid.org/).

### PR DESCRIPTION
**Pull request by jamienoss**
## Motivations:
 * ``zenodo.json`` is used by zenodo when a new DOI is created automatically when a new cut of the repo is released.
 * ``CITATION.cff`` is not used by ``zenodo`` when ``zenodo.json`` file also exists. So we nuke it.
 * The DOI to be cited is published by zenodo.org and therefore only the ``zenodo.json`` file matters.
 * The ``zenodo.json`` file allows for richer content, including:
   * **creators:** list of authors to be included in bibtex for DOI, these authors will incur a citation count when the DOI is referenced in published articles.
   * **contributors:** Others that contributed to the repo, including a specification of their role. These are not included in the citation for the DOI.
   * **references:**: List other articles/DOIs that the code uses/references. This allows repos generated from this base-template to reference the base-template DOI. This solves the need to include authors of the template in the contributors of the child repo.
  
## Zenodo docs on this topic:
 * [zenodo.jon docs](https://developers.zenodo.org/#representation) 
 * [zenodo docs on contributors vs creators](https://help.zenodo.org/docs/deposit/describe-records/contributors/)
Screenshot 2024-11-07 at 3 07 26 PM

## Caveats:
[zenodo.json](https://github.com/ssec-jhu/base-template/blob/main/.zenodo.json) is currently overloaded. It exists as both the citation of authors for the base-template DOI itself, i.e., https://zenodo.org/records/14052740 AND a simple reference of all possible SSEC authors (and importantly their orcid IDs) for the author of the child repo to prune from (saves new repo authors from having to find this elsewhere). Due to the latter use, all authors will be cited by the base-template DOI. We can either be ok with this (I am) or we can create a new auxiliary file with all orcid IDs in as reference.

This also resolves #159.

### CODE REVIEW

File: .zenodo.json [hunk 1]
```diff
@@ -51,6 +41,32 @@
...
+            "affiliation": "Scientific Software Engineering Center at JHU",
+            "name": "Edward Hunter",
+            "orcid": "0000-0002-6876-001X",
+            "type": "ProjectManager"
+        },
+        {
+            "affiliation": "Scientific Software Engineering Center at JHU",
+            "name": "Gerard Lemson",
+            "orcid": "0000-0001-5041-2458",
+            "type": "ProjectManager"
```
Comment (jamienoss) [inline .zenodo.json:55]: @glemson  Is this ok with you or would you like it changed?
The options are:
>ContactPerson, DataCollector, DataCurator, DataManager,Distributor, Editor, HostingInstitution, Producer, ProjectLeader, ProjectManager, ProjectMember, RegistrationAgency, RegistrationAuthority, RelatedPerson, Researcher, ResearchGroup, RightsHolder,Supervisor, Sponsor, WorkPackageLeader, Other

File: .zenodo.json [hunk 2]
```diff
@@ -51,6 +41,32 @@
...
+        },
+        {
+            "affiliation": "Scientific Software Engineering Center at JHU",
+            "name": "Galen Mayfield",
+            "orcid": "0009-0009-6192-8623",
+            "type": "ProjectManager"
+        }
+    ],
+    "grants": [
+        {"": ""}
```
Comment (jamienoss) [inline .zenodo.json:65]: @ehunter68 Would you be able to enquire with Schmidt about whether we can ref them as the grant source, please. It looks like zenodo accepts only OpenAIRE-supported grants for which I'm not sure if we're applicable. Here is the ``grants`` excerpt from the [zenodo docs](https://developers.zenodo.org/#representation):
>List of OpenAIRE-supported grants, which have funded the research for this deposition. Each array element is an object with the attributes:
>* id: grant ID.
>
>Example: [{'id':'283595'}] (European Commission grants only)
or funder DOI-prefixed: [{'id': '10.13039/501100000780::283595'}] (All grants, recommended)
>
>Accepted funder DOI prefixes:
Academy of Finland: 10.13039/501100002341
Agence Nationale de la Recherche: 10.13039/501100001665
Aligning Science Across Parkinson’s: 10.13039/100018231
Australian Research Council: 10.13039/501100000923
Austrian Science Fund: 10.13039/501100002428
Canadian Institutes of Health Research: 10.13039/501100000024
European Commission: 10.13039/501100000780
European Environment Agency: 10.13039/501100000806
Fundação para a Ciência e a Tecnologia: 10.13039/501100001871
Hrvatska Zaklada za Znanost: 10.13039/501100004488
Institut National Du Cancer: 10.13039/501100006364
Ministarstvo Prosvete, Nauke i Tehnološkog Razvoja: 10.13039/501100004564
Ministarstvo Znanosti, Obrazovanja i Sporta: 10.13039/501100006588
National Health and Medical Research Council: 10.13039/501100000925
National Institutes of Health: 10.13039/100000002
National Science Foundation: 10.13039/100000001
Natural Sciences and Engineering Research Council of Canada: 10.13039/501100000038
Nederlandse Organisatie voor Wetenschappelijk Onderzoek: 10.13039/501100003246
Research Councils: 10.13039/501100000690
Schweizerischer Nationalfonds zur Förderung der wissenschaftlichen Forschung: 10.13039/501100001711
Science Foundation Ireland: 10.13039/501100001602
Social Science Research Council: 10.13039/100001345
Templeton World Charity Foundation: 10.13039/501100011730
Türkiye Bilimsel ve Teknolojik Araştırma Kurumu: 10.13039/501100004410
UK Research and Innovation: 10.13039/100014013
Wellcome Trust: 10.13039/100004440

**Comment by jamienoss** [comment]
@ryanhausen Waiting on #152 for that file to be added, so I've deferred to #159.

**Comment by ryanhausen** [comment]
@jamienoss sounds good.

**Comment by jamienoss** [comment]
@ryanhausen This is ready for review, please. Note that I also resolved #159 in this PR. Thanks 👍

File: .zenodo.json [hunk 3]
```diff
@@ -51,6 +56,26 @@
...
+    ],
+    "references": [
+        "https://github.com/ssec-jhu/base-template 10.5281/zenodo.14052740."
+    ],
     "description": "Accelerating Software Development for Science Research",
     "license": "other-open",
```
Comment (ryanhausen) [inline .zenodo.json:80]: @jamienoss 

I think we can be more specific if we want with the license to match the license file. To match the LICENSE file we can change `"other-open"` to `"bsd-3-clause"`, which seems to be the id associated with the license:

https://zenodo.org/api/vocabularies/licenses?q=bsd-3-clause

No worries if you want to leave it as is.

**Review by ryanhausen — APPROVED** [review]

File: .zenodo.json (hunk 3 above)
Comment (jamienoss) [reply .zenodo.json:80]: Adding that now, thanks for finding that 👍

Comment (jamienoss) [reply .zenodo.json]: @ryanhausen back to you.

**Review by ryanhausen — APPROVED** [review]
lgtm!

**Pull request merged**